mmdet3d/ops/pointnet_modules/point_sa_module.py

```diff
@@ -24,6 +24,10 @@ class PointSAModuleMSG(nn.Module):
             Default: True.
         pool_mod (str): Type of pooling method.
             Default: 'max_pool'.
+        fps_mod (list[str]: Type of FPS method, valid mod
+            ['F-FPS', 'D-FPS', 'FS'], Default: ['D-FPS'].
```

Comment:
May tell the meaning of what does each of 'F-FPS', 'D-FPS', 'FS' means. Or you can use a more clear name rather than abbreviation.